projects/documentation/content/guides/adding-component.md

```diff
@@ -422,6 +422,14 @@ Information and examples about options
 of the component.
 -->
 
+### States
```

Comment:
This section was needed as we started documenting components with states like `disabled` and `pending`, so it has been added for consistency.